Makefile

```diff
@@ -11,7 +11,7 @@ all: deps gen build build_ui run
 rebuild: gen build
 
 test: verifiers
-	@GO111MODULE=on go test -mod=vendor -race -covermode=atomic -coverprofile=coverage.txt github.com/checkr/flagr/pkg/...
+	@GO111MODULE=on go test -mod=vendor -race -covermode=atomic -coverprofile=coverage.txt github.com/checkr/flagr/pkg/handler
```

Comment:
we still need to test basically all the packages under `pkd/...`

Reply:
whoops I thought I reverted that